app/controllers/coordinator_controller.rb

```diff
@@ -2,9 +2,19 @@ class CoordinatorController < ApplicationController
   before_action :auth
 
   def deliver
+    current_deliveries = Transaction.includes({recipient: [:recipient_profile]}, :donation).where(delivered_at: nil)
+    gon.deliveries = current_deliveries.as_json({include: [{recipient: {include: [:recipient_profile]}}, :donation]})
   end
 
   def schedule
+    gon.donations = []
+    gon.recipients = []
+    Interest.includes({recipient: [:recipient_profile]}, :donation).group_by(&:donation).each do |donation, interests|
+        gon.donations << donation
+        interests.each do |interest|
+            gon.recipients << interest.as_json({include: {recipient: {include: [:recipient_profile]}}})
```

Comment:
Line is too long. [103/80]<br>Redundant curly braces around a hash parameter.<br>Use 2 (not 4) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.